logins-sql/src/db.rs

```diff
@@ -0,0 +1,844 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use rusqlite::{Connection, types::{ToSql, FromSql}, Row};
+use std::time::SystemTime;
+use std::path::Path;
+use std::collections::HashSet;
+use error::*;
+use schema;
+use login::{LocalLogin, MirrorLogin, Login, SyncStatus, SyncLoginData};
+use sync::{self, ServerTimestamp, IncomingChangeset, Store, OutgoingChangeset, Payload};
+use update_plan::UpdatePlan;
+use util;
+
+pub const MAX_VARIABLE_NUMBER: usize = 999;
```

Comment:
There's a `rusqlite` function to get this at run-time; we should use it.  https://docs.rs/rusqlite/0.14.0/rusqlite/struct.Connection.html#method.limit